Document codind style exception for argument formatting in invocations (#5962)
SonarSource/sonar-dotnet: docs/coding-style.md

    someEnumerable.Where(x => x.Condition1
                              && x.Condition2);
    ```

>   * Exception from this rule: If the first argument is a lambda expression, the parameter name and the arrow token should go in the same line with the invocation.

Reviewer: To keep it simple and single-line
```suggestion
  * Exception from this rule: Lambda parameter name and arrow token should be on the same line as the invocation.
```
It intentionally ommits the "first lambda argument" part. That one is visible from example and not doable anywhere else. I'd be fine with having it as 2nd argument too, actually.
